03-a_review_of_r_modeling_fundamentals.Rmd

```diff
@@ -235,10 +256,10 @@ reg_fits <-
   trees2 %>%
   group_nest(group) %>% 
   mutate(
-    fit = map(data, ~ lm(formula(Volume ~ .), data = .x)),
-    tidied = map(fit, tidy),
-    glanced = map(fit, glance),
-    augmented = map(fit, augment)
+    fit = map(data, ~ lm(Volume ~ ., data = .x)),
```

Comment:
Depending on resolution of the conversation in the channel, possibly add `formula` back.